src/main/flight/pid.c

```diff
@@ -103,18 +106,18 @@ void resetPidProfile(pidProfile_t *pidProfile)
 
         .pidSumLimit = PIDSUM_LIMIT,
         .pidSumLimitYaw = PIDSUM_LIMIT_YAW,
-        .yaw_lpf_hz = 0,
-        .dterm_lpf_hz = 100,    // filtering ON by default
+        .yaw_lpf_hz = 20,
+        .dterm_lpf_hz = 40,    // filtering ON by default
         .dterm_notch_hz = 260,
         .dterm_notch_cutoff = 160,
         .dterm_filter_type = FILTER_PT1,
-        .dterm_filter_style = KD_FILTER_CLASSIC,
+        .dterm_filter_style = KD_FILTER_NOSP,
         .itermWindupPointPercent = 50,
         .vbatPidCompensation = 0,
         .pidAtMinThrottle = PID_STABILISATION_ON,
         .levelAngleLimit = 55,
         .setpointRelaxRatio = 100,
-        .dtermSetpointWeight = 0,
+        .dtermSetpointWeight = 127,
```

Comment:
Why are you setting this to 127 if you are effectivelly removing the dterm setpoint weight?

Reply:
I guess it would be good to cleanup the UI and remove the setpoint settings there too

Reply:
a left over from previous testing.
can be completely removed.
from the gui as well.